packages/vats/test/localchain.test.js

```diff
@@ -0,0 +1,183 @@
+// @ts-check
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { reincarnate } from '@agoric/swingset-liveslots/tools/setup-vat-data.js';
+import { E } from '@endo/far';
+import { M } from '@endo/patterns';
+import { getInterfaceOf } from '@endo/marshal';
+import { makeDurableZone } from '@agoric/zone/durable.js';
+import { AmountMath, AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { withAmountUtils } from '@agoric/zoe/tools/test-utils.js';
+import { buildRootObject as buildBankVatRoot } from '../src/vat-bank.js';
+import { prepareLocalChainTools } from '../src/localchain.js';
+
+/**
+ * @import {LocalChainAccount, LocalChainPowers} from '../src/localchain.js';
+ * @import {BridgeHandler, ScopedBridgeManager} from '../src/types.js';
+ */
+
+/** @type {import('ava').TestFn<ReturnType<makeTestContext>>} */
+const test = anyTest;
+
+const { fakeVomKit } = reincarnate({ relaxDurabilityRules: false });
+const provideBaggage = key => {
+  const root = fakeVomKit.cm.provideBaggage();
+  const zone = makeDurableZone(root);
+  return zone.mapStore(`${key} baggage`);
+};
+
+const makeTestContext = async t => {
+  const makeBridgeManager = async () => {
+    const zone = makeDurableZone(provideBaggage('mockBridgeManager'));
+    /** @type {undefined | ERef<BridgeHandler>} */
+    let bridgeHandler;
+
+    /** @type {ScopedBridgeManager} */
+    const bridgeManager = zone.exo('BridgeManager', undefined, {
+      async fromBridge(obj) {
+        t.is(typeof obj, 'string');
+      },
+      async toBridge(obj) {
+        switch (obj.type) {
+          case 'VLOCALCHAIN_ALLOCATE_ADDRESS': {
+            t.log('VLOCALCHAIN_ALLOCATE_ADDRESS', obj);
+            return 'agoricfoo';
+          }
+          default: {
+            t.is(obj, null);
+            return undefined;
+          }
+        }
+      },
+      initHandler(newHandler) {
+        bridgeHandler = newHandler;
+      },
+      setHandler(newHandler) {
+        bridgeHandler = newHandler;
+      },
+    });
+    return { bridgeManager, bridgeHandler };
```

Comment:
I don't see `bridgeHandler` being used. I think you just want to return `bridgeManager` itself.

Consider a TODO item here to adopt refactor to using https://github.com/Agoric/agoric-sdk/pull/9396 when it's available